x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.test.ts

```diff
@@ -331,32 +332,21 @@ describe('utils', () => {
       expect(gap?.asMilliseconds()).toEqual(moment.duration(1, 'minute').asMilliseconds());
     });
 
-    test('it returns null if given a previousStartedAt of null', () => {
+    test('it returns 0 if given a previousStartedAt of null', () => {
       const gap = getGapBetweenRuns({
         previousStartedAt: null,
-        interval: '5m',
+        intervalDuration: moment.duration(5, 'minutes'),
         from: 'now-5m',
         to: 'now',
         now: nowDate.clone(),
       });
-      expect(gap).toBeNull();
-    });
-
-    test('it returns null if the interval is an invalid string such as "invalid"', () => {
-      const gap = getGapBetweenRuns({
-        previousStartedAt: nowDate.clone().toDate(),
-        interval: 'invalid', // if not set to "x" where x is an interval such as 6m
```

Comment:
`getGapBetweenRuns` now takes an `intervalDuration` which has already been parsed, so this case is not needed.